master/static/srv/entrypoint.sh

```diff
@@ -64,4 +64,4 @@ test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}"
 # Do rendezvous last, to ensure all launch layers start around the same time.
 "$DET_PYTHON_EXECUTABLE" -m determined.exec.prep_container --rendezvous
 
-exec "$DET_PYTHON_EXECUTABLE" -m determined.exec.launch_autohorovod "$@"
+exec "$DET_PYTHON_EXECUTABLE" -m determined.launch "$@"
```

Comment:
Let's figure out the layout of naming we want for this.

What I see is that we have plans to make `determined.launch.autohorovod` a user-facing name, but `determined.launch` an internal name (i.e., only we call it, only once, and in the entrypoint.sh).  Perhaps we leave `determined.exec` for our internal executable scripts, like `determined.exec.launch` and we use `determined.launch.*` as user-facing names?

Reply:
agree with this scheme. moved `launch` into determined.exec